kms_reencrypt/filters/kms.py

```diff
@@ -14,6 +14,9 @@ def __init__(self, client_s3, client_kms, kms_key_id: str):
         self.client_s3 = client_s3
 
     def process(self, bucket: str, key: str) -> bool:
+        if key[-1] == "/":
+            return False
```

Comment:
Nit: Do you still want to retain this? Else https://github.com/dsaidgovsg/kms-reencrypt/commit/9500572fd09a9d871ec801cb0ab3a1db80433cd5 already removes the same code in executor side.

Reply:
Good catch! removed